suites/reef/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -58,10 +58,11 @@ tests:
       config:
         command: add
         id: client.1
-        node: node6
+        node: node4
```

Comment:
please get the client node directly in module code wise.

Reply:
Since persistent cache tests are running only in baremetal mero config, its fine to retain it here for now. We can change it when we are doing SDK implementation for persistent cache if required. Since this is unnecessary here for this test case.
In your test case, dm cache since it was running on both cephci and baremetal we had to do the change.